src/challenges/Day5.java

```diff
@@ -121,10 +121,14 @@ public Vent(Coordinate start, Coordinate end) {
     // matching values as the same. And defining a way to check object equivalence took super long.
     // Instead, we use prime numbers to try to make any Coordinate object have a unique key, and use that as a key for
     // the map
-    public int uniqueHash() {
+    public int hashCode() {
         return x * 7919 + y * 5897;
     }
 
+    public boolean equals(Object other) {
```

Comment:
Found out this is how to manipulate `HashSet` to consider two objects as the same, and not keep two different instances